Add unicode_literals to modules, states, and related tests

rallytime opened the pull request.

Commit:
Add unicode_literals to modules, states, and related tests

Commit:
Lint fix

terminalmage commented on salt/modules/kapacitor.py:
```diff
@@ -14,16 +14,18 @@
 from salt.utils.decorators import memoize
 
-LOG = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
```
Since we don't actually _use_ the log object in this file, we can remove lines 27 and 28.

terminalmage commented on salt/modules/virt.py:
```diff
@@ -404,23 +402,22 @@ def _qemu_image_create(vm_name,
                 __salt__['cmd.run'](
                     'qemu-img resize {0} {1}M'
-                    .format(img_dest, str(disk_size))
+                    .format(img_dest, six.text_type(disk_size))
```
`six.text_type` is unnecessary here since we're in a `.format()`.

terminalmage commented on salt/modules/virt.py:
```diff
@@ -440,10 +437,10 @@ def _qemu_image_create(vm_name,
                 __salt__['cmd.run'](
                     'qemu-img create -f {0} {1} {2}M'
-                    .format(disk_type, img_dest, str(disk_size))
+                    .format(disk_type, img_dest, six.text_type(disk_size))
```
`six.text_type` is unnecessary here since we're in a `.format()`.

terminalmage commented on salt/modules/virt.py:
```diff
@@ -710,27 +707,24 @@ def init(name,
-                          disk_name,
-                          diskp[0][disk_name].get('image', None)))
+        log.debug('%s image from module arguments will be used for disk "%s"'
+                  ' instead of %s', image, disk_name, diskp[0][disk_name].get('image', None))
```
`None` is implied if no argument is passed to a `dict.get()`. This doesn't necessarily need to be changed, though.

terminalmage commented on salt/states/kapacitor.py:
```diff
@@ -15,14 +15,15 @@
+import salt.utils.stringutils
 
-LOG = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
```
This log object is never used, lines 25 and 26 can be removed.

terminalmage reviewed (changes requested):
Just a few minor changes.

Commit:
Fixup small things from PR review comments

rallytime commented:
re-run py

Commit:
Fix state unit tests for kernelpkg

Commit:
Fix test errors created with unicode changes

- jboss_cli test: reduce test complication by not mixing raw and \u
- remove one test from test_junos: it's not really testing anything since it's heavily mocked
